sdk/python/feast/feature_view.py

```diff
@@ -176,3 +177,65 @@ def most_recent_end_time(self) -> Optional[datetime]:
         if len(self.materialization_intervals) == 0:
             return None
         return max([interval[1] for interval in self.materialization_intervals])
+
+    def run_reverse_field_mapping(
+        self, join_keys: List[str],
+    ) -> Tuple[List[str], List[str], str, Optional[str]]:
+        """
+        If a field mapping exists, run it in reverse on the join keys,
+        feature names, event timestamp column, and created timestamp column
+        to get the names of the relevant columns in the BigQuery table.
```

Comment:
I dont think we should be talking about BigQuery here. This is a general class. Unless we are referencing is as an example.